server/src/prisma/enum-adapter.ts

```diff
@@ -762,3 +762,26 @@ export const ActivityType = {
 } as const;
 
 export type ActivityType = (typeof ActivityType)[keyof typeof ActivityType];
+
+export const GroupTags = {
```

Comment:
Isn't this a GroupTagType?

Wouldn't it make more sense for a tag to be (groupId, type, index)?